src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -396,7 +396,6 @@ static bool containsOnlyOblivious(TypeSymbol type)
                     else
                     {
                         // Inconsistent accessibility: type '{1}' is less accessible than class '{0}'
-                        var lessVisibleTypeLocation = lessVisibleType.GetFirstLocation();
```

Comment:
As was already mentioned, we wouldn't want to use this here, since this situation doesn't indicate any problem with the definition of `lessVisibleType` (which may not be defined in source anyway), just with this particular usage.